packages/kbn-grid-layout/grid/use_grid_layout_state.ts

```diff
@@ -104,15 +122,22 @@ export const useGridLayoutState = ({
     const resizeSubscription = combineLatest([gridLayoutStateManager.gridDimensions$, accessMode$])
       .pipe(debounceTime(250))
       .subscribe(([dimensions, currentAccessMode]) => {
+        const currentRuntimeSettings = gridLayoutStateManager.runtimeSettings$.getValue();
```

Comment:
Good call to make this live. That will make it easier for us to change the column counts etc on the fly later.

Reply:
It was necessary for margin sizes, as well :)